### PR TITLE
use mutation observer to watch for body change & remove footer

### LINKED ISSUE #1: MMRA when user navigates to a post
Right now Medium is only Made Readable Again when a site first loads. This may leave readers with an un-great experience after they navigate to a post

### PR DESCRIPTION
**Pull request by kgdiem**
Resolves #1 by using a MutationObserver to run `makeReadable` when the body attributes change / a new page is loaded.

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -31,4 +37,6 @@ if (metaCheck != null && metaCheck.content == "Medium") {
 			hideDickbar();
 		}
 	});
+
+	observer.observe(document.querySelector('body'), target);
```
Comment (thebaer) [inline background.js]: Hey this looks like it'll be a great solution, but where is `target` coming from here? Getting an error here when I try to run it.

Comment (kgdiem) [reply background.js]: Oh whoops! That should be "config". I was dev'ing on WSL and forgot to copy it back before I committed. 

Will fix in a bit.

**Comment by kgdiem** [comment]
Updated!

**Comment by thebaer** [comment]
Awesome, thanks! Looks good.

**Pull request merged**